### PR TITLE
Add quantized Kinetics400 models

### PR DESCRIPTION
**Pull request by wuxun-zhang**
Added several quantized action recognition (2D) models for Kinetics400 datasets
cc @pengzhao-intel @xinyu-intel @bryanyzhu for review. Thanks.

### CODE REVIEW

**Comment by mli** [comment]
Job PR-1082-1 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1082/1/index.html 
Code coverage of this PR:  vs. Master:

**Comment by wuxun-zhang** [comment]
@bryanyzhu Currently we have already added several video action recognition 2D models which are targeting to the next gluon-cv release. We'll also continue to work on 3D models after some key optimizations are supported in MXNet.

File: scripts/action-recognition/test_recognizer.py [hunk 1]
```diff
@@ -244,7 +244,7 @@ def calibration(net, val_data, opt, ctx, logger):
     if isinstance(ctx, list):
         ctx = ctx[0]
     exclude_sym_layer = []
-    exclude_match_layer = []
+    exclude_match_layer = ['flatten']
```
Comment (xinyu-intel) [inline scripts/action-recognition/test_recognizer.py]: why exclude `flatten` here.

Comment (wuxun-zhang) [reply scripts/action-recognition/test_recognizer.py]: After double check, looks there is no need to explicitly specify `flatten` as being excluded.

**Comment by mli** [comment]
Job PR-1082-2 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1082/2/index.html 
Code coverage of this PR:  vs. Master:

**Pull request merged**